test/common/router/rds_impl_test.cc

```diff
@@ -182,6 +182,27 @@ TEST_F(RdsImplTest, UnknownCluster) {
                EnvoyException);
 }
 
+TEST_F(RdsImplTest, MaxRouteConfigName) {
+  const std::string config_json = R"EOF(
+  {
+    "rds": {
+      "cluster": "foo_cluster",
+      "route_config_name": "clusterwithareallyreallylongnamemorethanmaxcharsallowedbyschema"
+    },
+    "codec_type": "auto",
+    "stat_prefix": "foo",
+    "filters": [
+      { "name": "http_dynamo_filter", "config": {} }
+    ]
+  }
+  )EOF";
+
+  EXPECT_THROW(RouteConfigProviderUtil::create(parseHttpConnectionManagerFromJson(config_json),
```

Comment:
Can we do EXPECT_THROW_WITH_MESSAGE here?

Reply:
Actually, can I get rid of it? We have the same test in utility_test.cc

Reply:
Yup that's fine